### DISCUSSION TITLE
Make checkbox centered from the label.

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#9339] [Help]
Hi All,
How to make the checkbox become centered from the label ?
![image](https://github.com/filamentphp/filament/assets/6214964/8de4848a-af88-4235-92ee-cc996d14a39b)
here is the code :
```
                                Checkbox::make('check_1')->label('label check-1')
                                    ->inline(false)
                                    ->default(false),

                                Checkbox::make('check_2')->label('label check-2')
                                    ->inline(false)
                                    ->live()
                                    ->default(false),

                                Checkbox::make('check_3')->label('label check-3')
                                    ->inline(false)
                                    ->default(false),
```
I want to make like this image :
![image](https://github.com/filamentphp/filament/assets/6214964/0aada050-fb6f-4315-be79-f1f2f2175c77)
Thanks for help.

**Comment by leandrocfe** [comment]
Not sure because I'm on mobile. Maybe using extraAttributes 
`->extraAttributes(['class' => 'items-center justify-center'])`

**Reply by jigujigu** [reply]
Not working ...
any other ide ?

**Reply by Dr-PPSN** [reply]
It's been a while since you asked the question, but with `->extraAttributes(['style' => 'margin-left: 2rem;'])` you can achieve what you want, as shown in the image. Maybe you have to adjust the rem number but its one way...  
Example:  

<img width="181" height="60" alt="image" src="https://github.com/user-attachments/assets/2b2a5bac-0796-452d-a37b-1ee242dcd76c" />

With the code:
`
Checkbox::make('equipped')->label('Ausrüstung ausgerüstet')->inline(false)->default(false)->extraAttributes(['style' => 'margin-left: 4rem;']),
`
